catalog/ui/catalog-ui-search/src/main/webapp/js/model/QueryResponse.js

```diff
@@ -21,6 +21,16 @@ require('backboneassociations');
 var QueryResponseSourceStatus = require('js/model/QueryResponseSourceStatus');
 var QueryResultCollection = require('js/model/QueryResult.collection');
 
+let rpc = null
+
+if (window.WebSocket) {
+  const Client = require('rpc-websockets').Client
+  // Assume http means we are using the dev server, which means we should
```

Comment:
Is this a safe assumption?

Reply:
I am not sure. We could use `process.env.NODE_ENV !== 'production'` instead. So if it's a development build, use the default DDF port because you are probably being used with the dev server, otherwise use `location.port`.

Reply:
Actually I was able to get the dev server to proxy websocket connections, so I'll just remove the logic.